wiki/markdown/query.md

```diff
@@ -135,22 +131,21 @@ that [here](http://jmespath.org/tutorial.html#pipe-expressions).
 Example:
 
     // This match an object in the same way as in the previous example. And then filter
```

Comment:
In the section above: "## Query an aggregated object and filter it with specific key" The first sentence "It is possible to filter the object and return only values with specific key or path." should be updated to: "It is possible to filter the result from the query and return only the values which are of interest. This filter can be defined as a path in the aggregated object, or a specific key".

Reply:
done